stable/aws-calico/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 description: A Helm chart for installing Calico on AWS
 website: https://docs.aws.amazon.com/eks/latest/userguide/calico.html
 name: aws-calico
-version: 0.2.4
-appVersion: 3.8.1
+version: 0.3.0
+appVersion: 3.13.0
```

Comment:
I'd still prefer 3.13.4, so that it matches [the cni repo](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/config/v1.6/calico.yaml), but besides that the config changes look good.